.github/scripts/inject-token.sh

```diff
@@ -0,0 +1 @@
+echo '//npm.pkg.github.com/:_authToken=${NPM_TOKEN}' | cat - .npmrc > temp && mv temp .npmrc
```

Comment:
Is this adding the token to the last line of .npmrc?

Reply:
to the first line